### DISCUSSION TITLE
Use Elastic subscription

### DISCUSSION THREAD

**Discussion by lyr3051** [Axway-API-Management-Plus/apigateway-openlogging-elk#186] [Q&A] [answered]
Hi

If I want to use an Elastic subscription with this solution, do I have to configure it myself or do I need to contact you?

**Comment by cwiechmann** [comment, marked as answer]
You would have to contact Elastic to get a subscription from them:
https://www.elastic.co/pricing/

**Reply by lyr3051** [reply]
Hi @cwiechmann
So whether or not elastic subscription is used, this solution is applicable, right?

**Reply by cwiechmann** [reply]
Yes, see here: https://github.com/Axway-API-Management-Plus/apigateway-openlogging-elk#do-i-need-an-elastic-subscription

**Reply by lyr3051** [reply]
Thank you.
